### ISSUE TITLE
Like sometimes not registering?

### ISSUE THREAD

**Issue by hmpws** [litdevelopers/tinder#38]
I have had some issues where I liked a person and the exact same person shows up again later. I haven't done enough testing to see whether this occurs only to people at the top of the list after scrolling down (i.e. new recommendations may have been pulled). 

Does the user id and hash data get updated in the recommendation list if the same person is returned by Tinder? I understand that there is a specific user id and content id associated with each recommendation from Tinder, whether the content id actually affects the like command is conjecture though.

**Comment by eddiewang** [comment]
It's not the cleanest code. We created the project to learn React and JavaScript, and there are definitely bugs and optimizations we could add, but don't have time for at this stage in time. You caught one of the common bugs, it has to do with how we filter the recommendations (spoiler: it's pretty badly done).

**Comment by hmpws** [comment]
Yes, fair enough! I don't know enough about React nor JS unfortunately but tinkering is fun. I am guessing it is in this line from getRecommendations?

resolve(data.results.filter((each) => !each.is_brand));
